scripts.js

```diff
@@ -225,6 +224,95 @@ window.adobeid = {
   locale: window.blog.language,
 };
 
+/**
+ * Set up a click event on Region Picker
+ */
+function handleDropdownRegion() {
+  const currentLocale = window.blog.language;
+  const localeNames = {
+      'ko': 'Korea (한국어)',
+      'en': 'APAC (English)',
+      'en': 'USA (English)',
+      'en': 'UK (English)',
+  };
+
+  // Change locale value from Feds Region Picker Button
+  const FEDSregionPickerText = document.querySelector('.feds-regionPicker-text');
+  if (FEDSregionPickerText instanceof HTMLElement) {
+      FEDSregionPickerText.innerText = localeNames[currentLocale];
+  }
+
+  // Change Icon from Feds
+  const FEDSRegionPickerIcon = document.querySelector('.feds-regionPicker-icon');
+  if (FEDSRegionPickerIcon instanceof HTMLElement) {
+      FEDSRegionPickerIcon.innerHTML = `<svg xmlns="http://www.w3.org/2000/svg" width="19" height="19" viewBox="0 0 19 19">
```

Comment:
I think we can add the icon in the FEDS experience; changing the icon might not be needed.
cc @avanishgandhi